examples/benchmarks/tablebench/package.json

```diff
@@ -17,6 +17,7 @@
 	"types": "dist/index.d.ts",
 	"scripts": {
 		"bench": "mocha --timeout 999999 --perfMode --parentProcess --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js",
+		"bench:size": "mocha --timeout 999999 --perfMode --fgrep @Size",
```

Comment:
'bench:size' runs the size tests with '--perfMode'.

Without '--perfMode' the tests only create 100 rows instead of 10,000 to avoid the need for extended timeouts.